spidermon/contrib/scrapy/monitors.py

```diff
@@ -374,6 +380,53 @@ def test_check_if_field_coverage_rules_are_met(self):
         self.assertTrue(len(failures) == 0, msg=msg)
 
 
+@monitors.name("Jobs Comparison Monitor")
+class JobsComparisonMonitor(BaseScrapyMonitor):
+    """Check for a drop in scraped item count compared to previous jobs.
+
+    You need to set the number of previous jobs to compare, using ``SPIDERMON_JOBS_COMPARISON``.
+    The default is ``0`` which disables the monitor. We use the average of the scraped items count.
+
+    You can configure which percentage of the previous item count is the minimum acceptable, by
+    using the setting ``SPIDERMON_JOBS_COMPARISON_THRESHOLD``. We expect a float number between
+    ``0.0`` and ``1.0``. The default is ``0.8``.
+
+    You can filter which jobs to compare based on their states using the
+    ``SPIDERMON_JOBS_COMPARISON_STATES`` setting. The default value is ``("finished",)``.
+    """
+
+    @property
+    def client(self):
+        return hs
+
+    @monitors.name("Should not have a big drop in item count compared to previous jobs")
+    def test_if_item_count_from_previous_jobs_have_decreased(self):
+        # 1. get the number of jobs and check if monitor is enabled
+        number_of_jobs = self.crawler.settings.getint(SPIDERMON_JOBS_COMPARISON, 0)
```

Comment:
**suggestion** To avoid the monitor to be executed without the proper settings, you could do the validation of the existence of the settings and raise a [NotConfigured](https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/scrapy/monitors.py#L98)  exception when it is not. Doing that, we will receive a notification of the failure sooner.